test/test-framework/deploy/crds/cache.example.com_otherdummies_crd.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
It is not part of the scope of this CR. Could you please remove these files?

Reply:
ok